packages/toolkit/src/lib/markdown/MarkdownViewer.tsx

```diff
@@ -16,11 +16,13 @@ export const MarkdownViewer = ({
   markdown,
   skipHtml,
   style,
+  postFix,
 }: {
   className?: string;
   markdown: string;
   skipHtml?: boolean;
   style?: React.CSSProperties;
+  postFix?: React.ReactNode;
```

Comment:
@orangecoloured I would like to know more about this naming, it seems to me like a children but have more meaning to it, can you elaborate a bit?

Reply:
I had an “append” in mind. And it’s a verb. And we can also prepend. So I thought about prefixes and went with the postfix.

Reply:
I think you can name it suffix if that is the opposite of prefix 🤝